roles/openshift_facts/library/openshift_facts.py

```diff
@@ -629,6 +629,7 @@ def build_controller_args(facts):
                                   'cloudprovider')
     if 'master' in facts:
         controller_args = {}
+        controller_args['feature-gates'] = 'MountPropagation=false'
```

Comment:
Not sure if this was best place to add a controller argument. cc @sdodson

Reply:
Should be a list here: `['MountPropagation=false']`. In any case, I don't think its needed, since its being set in `l_node_kubelet_feature_flag_dict`

Reply:
It looks like we have moved node flags to openshift_node/defaults/main.yml but master still relies on openshift_facts and this is required by both master and node configs.